cli/ssh.go

```diff
@@ -509,6 +520,12 @@ func (r *RootCmd) ssh() *serpent.Command {
 			FlagShorthand: "e",
 			Value:         serpent.StringArrayOf(&env),
 		},
+		{
+			Flag:        "usage-app",
+			Description: "Specifies the usage app to use for workspace activity tracking.",
+			Env:         "CODER_SSH_USAGE_APP",
+			Value:       serpent.StringOf(&usageApp),
```

Comment:
Suggestion: use `serpent.EnumOf` here to restrict the allowed values.
Also hide this by default as it depends on an experiment being enabled.

Reply:
I've hidden the flag, good call on that. I've chosen not to use `serpent.EnumOf` because I don't want to hard error on invalid app names, just drop them and report as `ssh` in that case. That way if we add a new value to the enum we can update extensions without worrying about breaking old CLIs.

Reply:
Fair, `EnumOf` isn't as much of an issue if it's a hidden flag anyways.